springboard/templates/molecules.html

```diff
@@ -0,0 +1,123 @@
+{% import 'atoms.html' as atoms %}
+
+{% macro list_of_languages(view, languages, styling_classes) -%}
+<ul class="nav list_of_languages list-inline {{styling_classes}}" role="navigation">
+{% for lang_code, lang_name in languages %}
+  <li>
+    <a href={{view.request.route_url('locale_matched',  language=lang_code ) }}>{{lang_name}}</a>
+  </li>
+{% endfor %}
+	<li><a href="/spice/locale/change/">&gt;&gt;</a></li>
```

Comment:
Copy paste error from `unicore-cms`. This should be `{{view.request.route_url('locale_change')}}` (or something)